src/main/java/core/basesyntax/service/TransactionParser.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.Transaction;
+
+public class TransactionParser {
+    public Transaction parseLine(String line) {
+        String[] parts = line.split(",");
+        String operation = parts[0];
+        String fruitName = parts[1];
+        int quantity = Integer.parseInt(parts[2]);
```

Comment:
It's important to handle `NumberFormatException` when parsing the quantity. If `parts[2]` is not a valid integer, the method will throw an unhandled exception. You should catch this exception and throw a more informative `RuntimeException` indicating the problem with the input line.